### PR TITLE
Separate fields with a ,

### PR DESCRIPTION
**Pull request by mayhem**
I'm sorry for this rather large PR. mea culpa!

### CODE REVIEW

**Review by alastair — APPROVED** [review]
there's quite a bit that I'd like changed, but not enough time to go into detail. just merge for now and we can deal with the changes later

**Review by amCap1712 — APPROVED** [review]

File: brainzutils/metrics.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def set(metric_name: str, tags: Dict[str, str] = None, timestamp: int = None, **
         else:
             fields_list.append("%s=%s" % (k, str(v)))
 
-    fields = " ".join(fields_list)
+    fields = ",".join(fields_list)
```
Comment (MonkeyDo) [inline brainzutils/metrics.py:69]: This needs a full test suite !
For example, would this work with the greek question mark?

**Review by MonkeyDo — APPROVED** [review]
Preliminary comments, will come back to review the rest of the PR when I have 6 hours ahead of me.

File: brainzutils/metrics.py (hunk 1 above)
Comment (alastair) [reply brainzutils/metrics.py:69]: that being said, the tests did actually fail, so they're doing their job!

**Pull request merged**